lib/absinthe/schema/notation.ex

```diff
@@ -28,8 +28,7 @@ defmodule Absinthe.Schema.Notation do
     Module.register_attribute(__CALLER__.module, :absinthe_blueprint, accumulate: true)
     Module.register_attribute(__CALLER__.module, :absinthe_desc, accumulate: true)
     put_attr(__CALLER__.module, %Absinthe.Blueprint{schema: __CALLER__.module})
-    Module.put_attribute(__CALLER__.module, :absinthe_scope_stack, [:schema])
-    Module.put_attribute(__CALLER__.module, :absinthe_scope_stack_stash, [])
+    put_scope_stack(__CALLER__.module, [:schema])
```

Comment:
@binaryseed In master I added https://github.com/absinthe-graphql/absinthe/blob/master/lib/absinthe/schema/notation.ex#L1508 to deal with the fact that we needed two stacks to handle stack stashing for Absinthe Relay, we should unify these helpers.